binding/dune

```diff
@@ -0,0 +1,7 @@
+(library
+ (name bs_react_intl)
+ (public_name styled-ppx.bs-react-intl)
```

Comment:
?

Let's use `melange-react-intl.lib`, so that we can allow for `melange-react-intl.ppx` to exist, and ship both under the same package.

Reply:
I copied file from styled ppx and forgot to replace name 😬 

`melange-react-intl.lib` + `melange-react-intl.ppx` looks good 👍